src/pages/msme/SubmissionCompleted.tsx

```diff
@@ -59,6 +63,8 @@ function SubmissionCompleted() {
                 'In the meantime if you have any questions, you can reach out to member of the Open Contracting Partnership team at: credere@open-contracting.org.',
               )}
             </Text>
+            <Text className="mb-8">{t('Thank you for counting with us')}</Text>
```

Comment:
I don't know the meaning of this expression.

Reply:
It was Google translated from "Gracias por contar con nosotros", "Thank you very much for choosing us." is more accurate I guess. I didn't pay much attention to it as we are not using the EN versions anyway, but I can change it